xbmc/utils/EGLUtils.cpp

```diff
@@ -667,13 +667,16 @@ void CEGLContextUtils::SetDamagedRegions(const CDirtyRegionList& dirtyRegions)
   }
   else
   {
-    EGLint height = eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height);
+    EGLint height;
```

Comment:
Please initialize to 0, so when the below call fails, we still have a valid value.

Reply:
Yeah, sorry about that. I've defaulted to a height of 1080, if that's all right.